.yaspeller.json

```
@@ -3,6 +3,7 @@
   "ignoreUrls": true,
   "checkYo": true,
   "dictionary": [
+    "андройдов",
```

```suggestion
    "андроидов",
```

@tachisis нормативно же андроидов, без «й»?